CHANGELOG.md

```diff
@@ -12,4 +12,11 @@ the following form: YYYY.0M.0D.
 
 - Designed a musicPlaylist component
 - Designed a shoppingCart component
-- Designed a library component
\ No newline at end of file
+- Designed a library component
+
+## 2024.10.03
```

Comment:
Looks good. I might recommend that you put the latest changes at the top of the file (i.e., reverse chronological order).